staking/tests/config.ts

```diff
@@ -363,6 +369,58 @@ describe("config", async () => {
         pythTokenListTime: null,
         agreementHash: getDummyAgreementHash(),
         mockClockTime: new BN(30),
+        poolAuthority,
```

Comment:
In this file you should add this to the `someone else tries to access admin methods` as well

Reply:
Done